source/config.js

```diff
@@ -11,6 +11,10 @@ var defaultConfig = {
 
   // The base URL ungit will be accessible from.
   urlBase: "http://localhost",
+  
+  // The URL root path under which ungit will be accesible.
+  // Either blank or starting with a slash and no trailing slash
```

Comment:
I'm not sure if I have explained correctly, if we do `url.resolve(urlBase + ':' + port, rootPath)` at the later part of this file, we don't need validation for checking prefix, postfix slashes within rootPath or port as it is handled via url package.  

What I mean by additional validation is the character encoding and non alphabet characters and such....